cmd/bundle/validate.go

```diff
@@ -18,6 +18,9 @@ import (
 )
 
 func renderJsonOutput(cmd *cobra.Command, b *bundle.Bundle) error {
+	if b == nil {
```

Comment:
Why would it be nil in the first place? From what I thought it's only nil when error is return from bundle configure function, maybe we're missing handling the error somewhere?